demo/src/screens/CheckoutScreen/CheckoutScreen.js

```diff
@@ -1,19 +1,55 @@
 import React from 'react';
+import {Container, Col, Row, Button} from 'react-bootstrap';
+import {UserInfoForm} from '../../components/UserInfoForm/UserInfoForm.js';
+import {useHistory} from 'react-router-dom';
+import './CheckoutScreen.css';
 import {MiniCart} from '../../components/MiniCart/MiniCart.js';
-import {Container, Col, Row} from 'react-bootstrap';
 
 /**
- * @return {!JSX} Page component for where a user can purchase
- *     the items they've selected.
+ * The ID for the form the user will fill out on this page.
+ * @const
+ * @type {string}
```

Comment:
nit: I don't *think* you need @type here... just @const {string}

Reply:
Done